### PR TITLE
fix: Set fixed maxWidth of the cron schedule modal

### PR DESCRIPTION
**Pull request by codemaster08240328**
### SUMMARY

When we select `year`/`month` cases from cron schedule modal, the modal width is changed dynamically. 

We need to set fixed max width of the modal. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before: 
<a href="https://www.loom.com/share/1729d2ab62bf4f499843433910d75b06">
    <p>FCC New Coder Survey 2018 - 1 April 2022 - Watch Video</p>
    
  </a>
After:
<a href="https://www.loom.com/share/b19c6e4904b5448ca8623955747fdddd">
    <p>FCC New Coder Survey 2018 - 1 April 2022 - Watch Video</p>
    
  </a>

### TESTING INSTRUCTIONS

1. Open a dashboard or a chart
2. Click the calendar icon

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by diegomedina248 — APPROVED** [review]

**Comment by yousoph** [comment]
> Checking in on the intended UX here. In the After video, we still see the width of the modal changing... did we want to make it some sensible fixed/min width so that it doesn't push and pull at all based on content changes? @yousoph @eschutho

Yeah - the min width should be set to what we see in the second half of the "After" video so it doesn't jump when switching to month and year selections

**Comment by codemaster08240328** [comment]
resolved comment @rusackas 
[FCC New Coder Survey 2018 - 21 April 2022 - Watch Video](https://www.loom.com/share/bbf1f2012aba4f088e25c2d01955ff08)

File: superset-frontend/src/components/ReportModal/styles.tsx [hunk 1]
```diff
@@ -71,6 +71,7 @@ export const StyledScheduleTitle = styled.div`
 
 export const StyledCronPicker = styled(CronPicker)`
   margin-bottom: ${({ theme }) => theme.gridUnit * 3}px;
+  width: 450px;
```
Comment (rusackas) [inline superset-frontend/src/components/ReportModal/styles.tsx:74]:
```suggestion
  width: ${({ theme }) => theme.gridUnit * 120}px;
```
Is 480px too much? This seems like a nice round-ish number of gridUnits (divisible by 3 or 4)

**Comment by codemaster08240328** [comment]
@rusackas , the test environment above you created, doesn't have report/alert config on. So we can't see this modal unfortunately there.

**Review by rusackas — APPROVED** [review]

**Pull request merged**